.github/workflows/receive-fork-review.yml

```diff
@@ -6,15 +6,15 @@ name: Receive Fork Review
 
 on:
   pull_request_review:
-    branches: [main]
```

Comment:
`pull_request_review` doesn't have a `branches` filter unlike `pull_request`, so moved the logic below.